src/pages/registration/contributors_tab/Authors.tsx

```diff
@@ -0,0 +1,130 @@
+import deepmerge from 'deepmerge';
+import { FieldArrayRenderProps, FormikProps, useFormikContext, move } from 'formik';
+import React, { FC, useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { setNotification } from '../../../redux/actions/notificationActions';
+import { Authority } from '../../../types/authority.types';
+import { Contributor, emptyContributor, UnverifiedContributor } from '../../../types/contributor.types';
+import { NotificationVariant } from '../../../types/notification.types';
+import { BackendTypeNames } from '../../../types/publication_types/commonRegistration.types';
+import { Registration } from '../../../types/registration.types';
+import { overwriteArrayMerge } from '../../../utils/formik-helpers';
+import AuthorTable from './components/AuthorTable';
+import AddIcon from '@material-ui/icons/Add';
+import styled from 'styled-components';
+import { Button } from '@material-ui/core';
+import AddContributorModal from './AddContributorModal';
+
+const StyledAddAuthorButton = styled(Button)`
+  margin-top: 1rem;
+`;
+
+interface AuthorsProps extends Pick<FieldArrayRenderProps, 'push' | 'replace'> {}
+
+const Authors: FC<AuthorsProps> = ({ push, replace }) => {
+  const { t } = useTranslation('registration');
+  const dispatch = useDispatch();
+  const { values, setValues }: FormikProps<Registration> = useFormikContext();
+  const {
+    entityDescription: { contributors },
+  } = values;
+  const [openContributorModal, setOpenContributorModal] = useState(false);
+  const [unverifiedAuthor, setUnverifiedAuthor] = useState<UnverifiedContributor | null>(null);
+  const orderedAuthors = [...contributors].map((contributor, index) => ({ ...contributor, sequence: index + 1 }));
```

Comment:
```suggestion
  const orderedAuthors = contributors.map((contributor, index) => ({ ...contributor, sequence: index + 1 }));
```
tror backend nå bare baserer seg på `sequence` for rekkefølge (ikke rekkefølgen i arrayet), men jeg syns de bør være lik i array også, så enig i hvordan det er her

Reply:
vi kunne evt tatt en `.sort()` på `sequence` før vi mapper over med oppdatert index, slik at vi er sikre på at to stk ikke har samme sequence

Reply:
tenkte at jeg ikke skulle modifisere `contributors` direkte, men heller modifisere kopien. Men det spiller kanskje ikke så stor rolle når det er FormikValues uansett. tror ikke det er nødvendig med sort når vi gjør det på den måten der altså. Vi bruker jo `index`, og den er alltid unik. (testet med en publikasjon med to forfattere som hadde sequence: 1 fra backend)

Reply:
contributors blir uansett ikke oppdatert uten en setValues

litt usikker på hvordan backend gjør det med sequence, for hvis de bare baserer seg på sequence kan det bli feil, feks
```
[
{seuence: 2, (...)},
{seuence: 1, (...)}
]
```
men det vil være rart da, så vi kan anta at sequence og index matcher, og heller lage en bug til backend hvis det blir fuckups en gang

Reply:
ja, _noen_ krav må vi hvertfall stille til backend 😛 egentlig synes jeg denne sorteringa burde blitt gjort på backend i utgangspunktet, menmen